<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsMeshTransformCoordinatesDialogBase</class>
 <widget class="QDialog" name="QgsMeshTransformCoordinatesDialogBase">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QgsExpressionLineEdit" name="mExpressionEditX" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QgsExpressionLineEdit" name="mExpressionEditZ" native="true"/>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QDialogButtonBox" name="mButtonHelp">
       <property name="standardButtons">
        <set>QDialogButtonBox::Help</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mButtonPreview">
       <property name="text">
        <string>Preview transform</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mButtonApply">
       <property name="text">
        <string>Apply transform</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="mCheckBoxX">
     <property name="text">
      <string>X coordintate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="mCheckBoxY">
     <property name="text">
      <string>Y coordinate</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="mCheckBoxZ">
     <property name="text">
      <string>Z value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QgsExpressionLineEdit" name="mExpressionEditY" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsExpressionLineEdit</class>
   <extends>QWidget</extends>
   <header>qgsexpressionlineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
